src/test/integration/oauth-test.js

```diff
@@ -385,7 +386,7 @@ describe('oauth/', function() {
     expect(jwt).toBeTruthy();
 
     const err = await pFinal(chai.request(server)
-      .get(Constants.THINGS_PATH)
+      .get(`${Constants.THINGS_PATH}/${TEST_THING.id}`)
```

Comment:
This test shouldn't need modification. An authorization code should not be able to access `/things`. The change I suggested above should fix this behavior.